docs/basic-features/environment-variables.md

```diff
@@ -152,7 +152,7 @@ export default async () => {
 
 ## Environment Variable Load Order
 
-Depending on the environment (as set by `NODE_ENV`), Environment Variables are loaded from the following sources in top-to-bottom order. In all environments, the existing `env` is not overridden by following sources:
+Depending on the environment (as set by `NODE_ENV`), Environment Variables are defined from the following sources with descending priority. In all environments, the existing system `env` has the highest priority and is not overridden.
```

Comment:
```suggestion
Depending on the environment (as set by `NODE_ENV`), Environment Variables are defined from the following sources with descending priority. This means that the lowest priority (4) is loaded first, then 3 will override, then 2 will override, and finally 1 has the highest priority and will override all others. In all environments, the existing system `env` has the highest priority and is not overridden.
```

I think we need to explain overriding here because "priority" isn't enough to explain it.